docker/Dockerfile

```diff
@@ -83,6 +83,8 @@ RUN apt-get update \
 		python3-wheel \
 		# Used by managarm
 		python3-protobuf \
+		# Used by libxcb
+		python3-xcbgen \
```

Comment:
By how much does the rootfs size increase if we pull this package?

Reply:
Uhhhh, unsure, but I'm pretty sure it's small. I'll check when I have time and remember how to do that.

Reply:
Less than 10 megabytes, I don't have anything more accurate than that. Note that this is the output of `docker images`, so this is uncompressed.